server/api/router.go

```diff
@@ -46,6 +46,7 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	router.HandleFunc("/api/v1/schedulers/{name}", schedulerHandler.Delete).Methods("DELETE")
 
 	router.Handle("/api/v1/cluster", newClusterHandler(svr, rd)).Methods("GET")
+	router.HandleFunc("/api/v1/raft/bootstrap/time", newClusterHandler(svr, rd).GetRaftClusterBootstrapTime).Methods("GET")
 
```

Comment:
I  think /cluster/raft is ok, which will return the whole raft cluster information including the bootstrap time.